cargo-pgx/default.nix

```diff
@@ -1,14 +1,8 @@
 { lib
 , naersk
 , hostPlatform
-, fetchFromGitHub
-, postgresql_10
-, postgresql_11
-, postgresql_12
-, postgresql_13
 , pkg-config
 , openssl
-, rustfmt
```

Comment:
Actually, wait. I know they look unused but can you reduce this to the most conservative change?

Reply:
Sure, should I just remove 0f0a1618e2dca0f4b9c3f4057da841c6f4df3358 from this PR or add another commit that only removes `fetchFromGitHub`?

Reply:
Pushing another commit is fine.

Reply:
Apologies, I am on a timer.